features/crud.feature

```diff
@@ -16,7 +16,9 @@ Feature: Create-Retrieve-Update-Delete
           "value1",
           "value2"
         ]
-      }
+      },
+      "dummyFloat": 1.2,
+      "dummyPrice": 1.2
```

Comment:
Yeah, forgot that the precision loss is also due to php... Would you be able to tell me where I may fix this?